src/targets/gpu/target.cpp

```diff
@@ -84,6 +84,9 @@ MIGRAPHX_DECLARE_ENV_VAR(MIGRAPHX_ENABLE_CK)
 
 std::vector<pass> target::get_passes(migraphx::context& gctx, const compile_options& options) const
 {
+    size_t free_mem = 0;
+    auto status     = hipMemGetInfo(&free_mem, nullptr);
+    std::cout << "Free memory: " << free_mem << " status: " << status << std::endl;
```

Comment:
These debug prints shouldn't be in the `get_passes`.

Reply:
Removed, thanks